pkg/controller/controller.go

```diff
@@ -19,15 +18,16 @@ import (
 
 	"github.com/jetstack/version-checker/pkg/client"
 	"github.com/jetstack/version-checker/pkg/controller/scheduler"
+	"github.com/jetstack/version-checker/pkg/controller/search"
 	"github.com/jetstack/version-checker/pkg/metrics"
 	"github.com/jetstack/version-checker/pkg/version"
 )
 
 const (
-	numWorkers = 5
+	numWorkers = 10
```

Comment:
How did you decide on 10 workers for the workqueue specifically, rather than 5 as before?  Would it be worth to be able to pass an option for this?

Reply:
I found it a bit better performance wise for aggregating the versions faster. We're not doing anything too crazy, just making a lot of http GET requests. I think until this becomes an issue, we can leave as is and think about making it configurable later.